src/common/serializers.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+
+from . import models
+from account.models import Profile
+
+
+class AbstractCommentSerializer(serializers.ModelSerializer):
+    vote_count = serializers.SerializerMethodField()
+    user_vote = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.AbstractComment
+        fields = ('text', 'owner', 'show_username', 'vote_count', 'user_vote')
+        read_only_fields = ('created_at', 'updated_at')
+
+    def get_vote_count(self, obj):
+        upvotes = obj.votes.upvoters.count()
+        downvotes = obj.votes.downvoters.count()
+        return upvotes - downvotes
+
+    def get_user_vote(self, obj):
+        try:
+            current = self.context['request'].user.username
+            if obj.votes.upvoters.filter(
+                    user__username=current
```

Comment:
4 space

Reply:
Fixed in https://github.com/DevTeamSCH/vikoverflow-backend/pull/11/commits/e63e5aa44011d4d34b60f42930ed1ac657d5aa48.